src/dynamoTable.py

```diff
@@ -0,0 +1,34 @@
+import boto3
+from boto3.dynamodb.conditions import Key, Attr
+
+class DynamoTable(object):
+    
+    """Initialize table
+       Can be buildings, machines, users
+    """
+    def __init__(self, tableType):
+        allowedTableTypes = ['Buildings', 'Machines', 'Users']
+        if tableType not in allowedTableTypes:
+            raise ValueError('Invalid table type! Must be one of: {}'.format(allowedTableTypes))
+        dynamodb = boto3.resource('dynamodb')
+        self.table = dynamodb.Table(tableType)
+
+    def putItem(self, Info):

```

Comment:
Minor nit, you don't need like `Item` or `Table` in the method names. `get(...)`, `put(...)`, `scan()` should be good.

For the scan method, you could have a default param `filterExpression=None`, then if it's not None you use the expression.